DEVELOPER.md

```diff
@@ -1,188 +1,84 @@
 - [Developer quickstart](#developer-quickstart)
```

Comment:
add section on using own IDE, maybe give options to disable certain docker-compose containers (ie. profiles)